guides/source/5_1_release_notes.md

```diff
@@ -228,90 +228,140 @@ can generate form tags based on URLs, scopes or models.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+Please refer to the [Changelog][action-pack] for detailed changes.
 
 Action View
 -------------
 
-Please refer to the [Changelog][action-view] for detailed changes.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+Please refer to the [Changelog][action-view] for detailed changes.
 
 Action Mailer
 -------------
 
-Please refer to the [Changelog][action-mailer] for detailed changes.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+*   Allowed setting custom content type when attachments are included
+    and body is set inline.
+    ([Pull Request](https://github.com/rails/rails/pull/27227))
+
+*   Allowed passing lambdas to the default hash.
+    ([Commit](https://github.com/rails/rails/commit/1cec84ad2ddd843484ed40b1eb7492063ce71baf))
+
+*   Added support for parameterized invocation of mailers to share before filters and defaults
+    between different mailer actions.
+    ([Commit](https://github.com/rails/rails/commit/1cec84ad2ddd843484ed40b1eb7492063ce71baf))
+
+*   Passed the incoming arguments to the mailer action to `process.action_mailer` event under
+    `args` key.
+    ([Pull Request](https://github.com/rails/rails/pull/27900))
+
+Please refer to the [Changelog][action-mailer] for detailed changes.
 
 Active Record
 -------------
 
-Please refer to the [Changelog][active-record] for detailed changes.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+*   Skipped comments in the output of `mysqldump` command by default.
+    ([Pull Request](https://github.com/rails/rails/pull/23301))
+
+*   Fixed `ActiveRelation#count` to use Ruby's `Enumerable#count` for counting
+    records when a block is passed as argument instead of silently ignoring the
+    passed block.
+    ([Pull Request](https://github.com/rails/rails/pull/24203))
+
+*   Pass `"-v ON_ERROR_STOP=1"` flag with `psql` command to not suppress SQL errors.
+    ([Pull Request](https://github.com/rails/rails/pull/24773))
+
+Please refer to the [Changelog][active-record] for detailed changes.
 
 Active Model
 ------------
 
-Please refer to the [Changelog][active-model] for detailed changes.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+Please refer to the [Changelog][active-model] for detailed changes.
 
 Active Job
 -----------
 
-Please refer to the [Changelog][active-job] for detailed changes.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+Please refer to the [Changelog][active-job] for detailed changes.
 
 Active Support
 --------------
 
-Please refer to the [Changelog][active-support] for detailed changes.
-
 ### Removals
 
 ### Deprecations
 
-### Notable changes
+### Notable Changes
+
+*   Added `Module#delegate_missing_to` to delegate missing methods from a class to
```

Comment:
I'd rather say "to delegate `method_missing` and `respond_to_missing?` to a proxy object".